apps/ui/src/views/Space/Editor.vue

```diff
@@ -433,50 +461,101 @@ watchEffect(() => {
     <div class="flex items-stretch md:flex-row flex-col w-full md:h-full">
       <div class="flex-1 grow min-w-0">
         <UiContainer class="pt-5 !max-w-[710px] mx-0 md:mx-auto s-box">
-          <MessageVotingPower
-            v-if="propositionPower"
-            class="mb-4"
-            :voting-power="propositionPower"
-            action="propose"
-            @fetch-voting-power="handleFetchPropositionPower"
-          />
           <UiAlert
             v-if="
-              propositionPower &&
-              spaceType === 'default' &&
-              proposalLimitReached
+              !space.turbo &&
+              unsupportedProposalNetworks.length &&
+              !proposal?.proposalId
             "
             type="error"
             class="mb-4"
           >
-            <span
-              >Please verify your space to publish more proposals.
-              <a
-                :href="VERIFIED_URL"
-                target="_blank"
-                class="text-rose-500 dark:text-neutral-100 font-semibold"
-                >Verify space</a
-              >.</span
-            >
+            <div>
+              You cannot create proposals. This space is configured with
+              non-premium networks (<template
+                v-for="(n, i) in unsupportedProposalNetworks"
+                :key="n.key"
+              >
+                <b>{{ n.name }}</b>
+                <template
+                  v-if="
+                    unsupportedProposalNetworks.length > 1 &&
+                    i < unsupportedProposalNetworks.length - 1
+                  "
+                  >,
+                </template> </template
+              >). Change to a
+              <AppLink
+                to="https://help.snapshot.box/en/articles/10478752-what-are-the-premium-networks"
+                >premium network
+                <IH-arrow-sm-right class="inline-block -rotate-45" />
+              </AppLink>
+              or upgrade
+              <template
+                v-for="(n, i) in unsupportedProposalNetworks"
+                :key="n.key"
+              >
+                <b>{{ n.name }}</b>
+                <template
+                  v-if="
+                    unsupportedProposalNetworks.length > 1 &&
+                    i < unsupportedProposalNetworks.length - 1
+                  "
+                  >,
+                </template>
+              </template>
+              to continue.
```

Comment:
how about this, as we already have this list before 
![image](https://github.com/user-attachments/assets/c54ad0de-78a8-4e17-a3ea-8b3cca364cf3)

```suggestion
              or upgrade networks to continue.
```

Reply:
@bonustrack , wdyt of this shorter message ?

Reply:
Works for me.